Rename and describe headers in output files (#14)
VIB-PSB/ksrates: doc/source/input_output.rst

    Files:

        *   Raw rate-adjustment results for each trio (``adjustment_table_species_all.tsv``). Tabular format.

>             Each row shows the result for a species pair (column 2 ``Focal_Species`` and 3 ``Sister_Species``) diverging at a certain node (column 1 ``Node``) and adjusted with the outgroup in column 3 ``Out_Species``. The rate-adjusted mode with associated standard deviation are given in column 4 ``Adjusted_Mode`` and 5 ``Adjusted_Mode_SD``; for comparison it is provided the unadjusted mode with associated standard deviation (column 6 ``Original_Mode`` and 7 ``Original_Mode_SD``). The branch-specific *K*:sub:`S` contributions for the divergent species pair are listed in column 8 ``Ks_Focal`` and 9 ``Ks_Sister``; the *K*:sub:`S` contribution of the outgroup is listed in column 10 ``Ks_Out``.

Reviewer: ```suggestion
            Each row shows the result for a species pair (column 2 ``Focal_Species`` and 3 ``Sister_Species``) diverging at a certain node (column 1 ``Node``) and adjusted with the outgroup in column 3 ``Out_Species``. The rate-adjusted mode with associated standard deviation are given in column 4 ``Adjusted_Mode`` and 5 ``Adjusted_Mode_SD``; for comparison the unadjusted original mode with associated standard deviation is provided in column 6 ``Original_Mode`` and 7 ``Original_Mode_SD``. The branch-specific *K*:sub:`S` contributions for the divergent species pair are listed in column 8 ``Ks_Focal`` and 9 ``Ks_Sister``; the *K*:sub:`S`distance of the outgroup to the divergence event of the species pair is listed in column 10 ``Ks_Out``.
```

Author: I was afraid that "Ks distance" could be understood as an actual Ks estimate, while "contributions" indicates only the substitutions on the single branch.

Reviewer: I understand but here it's not really a contribution, Ks_Out is the distance C to divergence of A and B (what we call O) not to the divergence of A and C or B and C.

Author: Ooh, I see... Let's then leave distance!

            .. figure:: _images/adj_table_all.png
                :align: center
                :width: 800

                Raw rate-adjustment results on a divergent pair using four outgroups.

        *   Final rate-adjustment results for each divergent species pair after finding a consensus value in case of multiple outgroups (``adjustment_table_species.tsv``). Tabular format.
        

>             Each row shows the result for a species pair (column 2 ``Focal_Species`` and 3 ``Sister_Species``) diverging at a certain node (column 1 ``Node``). Columns 4-7 report the consensus obtained by taking the *mean* over raw results: rate-adjusted mode with standard deviation in column 4 ``Adjusted_Mode_Mean`` and 5 ``Adjusted_Mode_SD_Mean``, branch-specific *K*:sub:`S` contributions for the divergent species pair in column 6 ``Ks_Focal_Mean`` and 7 ``Ks_Sister_Mean``. Columns 8-11 report the consensus obtained when considering only the *best outgroup*: rate-adjusted mode with standard deviation in column 8 ``Adjusted_Mode_Best`` and 9 ``Adjusted_Mode_SD_Best``, *K*:sub:`S` contributions for the divergent species pair in column 10 ``Ks_Focal_Best`` and 11 ``Ks_Sister_Best``. For comparison it is provided the unadjusted mode with associated standard deviation (column 12 ``Original_Mode`` and 13 ``Original_Mode_SD``).

Reviewer: ```suggestion
            Each row shows the result for a species pair (column 2 ``Focal_Species`` and 3 ``Sister_Species``) diverging at a certain node (column 1 ``Node``). Columns 4--7 report the consensus obtained by taking the *mean* of multiple outgroups (if available): rate-adjusted mode with standard deviation in column 4 ``Adjusted_Mode_Mean`` and 5 ``Adjusted_Mode_SD_Mean``, branch-specific *K*:sub:`S` contributions for the divergent species pair in column 6 ``Ks_Focal_Mean`` and 7 ``Ks_Sister_Mean``. Columns 8-11 report the consensus obtained when considering only the *best outgroup*: rate-adjusted mode with standard deviation in column 8 ``Adjusted_Mode_Best`` and 9 ``Adjusted_Mode_SD_Best``, *K*:sub:`S` contributions for the divergent species pair in column 10 ``Ks_Focal_Best`` and 11 ``Ks_Sister_Best``. For comparison the unadjusted original mode with associated standard deviation is provided in column 12 ``Original_Mode`` and 13 ``Original_Mode_SD``.
```
